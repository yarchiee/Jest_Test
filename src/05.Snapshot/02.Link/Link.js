export default function Link(props) {
  return <a href={props.page}>{props.children}</a>;
}
