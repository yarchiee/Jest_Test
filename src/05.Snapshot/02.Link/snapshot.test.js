import TestRenderer from "react-test-renderer";
import Link from "./Link";

it("snapshot of button", () => {
  const tree = TestRenderer.create(
    <Link
      page="https://www.facebook.com/%22%3EFacebook"
      children={"facebook"}
    ></Link>
  ).toJSON();
  console.log(tree);
  expect(tree).toMatchSnapshot();
});
