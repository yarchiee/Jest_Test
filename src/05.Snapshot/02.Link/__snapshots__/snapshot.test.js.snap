// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of button 1`] = `
<a
  href="https://www.facebook.com/%22%3EFacebook"
>
  facebook
</a>
`;
